internal/cmprefimpl/cmpgeos/checks.go

```diff
@@ -817,6 +834,11 @@ func checkDCELOp(
 		return nil
 	}
 
+	got, err := op(g1, g2)
```

Comment:
Previously the skip list was only checked after the simplefeatures operation was executed. This was fine when the skip list was just for skipping comparison of the results. However, now it's also skipping geometries that cause the operation to fail outright. So the skip check has to go before the actual operation is executed.